jenkins-build.sh

```diff
@@ -309,7 +315,12 @@ fi
 
 if [ $skip_build -eq 0 ]
 then
-    ./rebuildall.sh --no-motr-rpm --use-build-cache $valgrind_flag
+    extra_opts=""
+    if [ $skip_ut_build -ne 0 ]
```

Comment:
Can we add an option to 'run' only UTs and skip STs?
It will help and speed up https://github.com/Seagate/cortx-s3server/pull/466 (EOS-15345)

Reply:
@nileshgovande -- I added this change few days ago while working on my fix.  I am not sure when I can have time to implement "run only UTs".  But this PR can be delivered this year as it is, and will help in future.  I suggest let us merge it as is, and take this "run only UT" as a separate fix.

Reply:
Added the option -- please review and let me know if its ok.  Option is --skip_ut_run and --skip_st_run.